lighthouse-core/report/styles/report.css

```diff
@@ -29,10 +29,10 @@ span, div, p, section, header, h1, h2, li, ul {
   --text-color: #222;
   --secondary-text-color: #606060;
   --accent-color: #3879d9;
-  --poor-color: #e81e1b;
-  --good-color: #008a07;
-  --average-color: #9f6d00;
-  --warning-color: #947200;
+  --poor-color: #e53935; /* md red 600 */
+  --good-color: #43a047; /* md green 600 */
+  --average-color: #ef6c00; /* md orange 800 */
+  --warning-color: #757575; /* md grey 600 */
```

Comment:
Right. My thinking was that colors should signify level of passing. Since we moved the warning icon to orange, it's felt like mixed messaging. e.g. orange for an audit error and orange for a 50/100 audit is kinda confusing. If anything, it might make more sense to make the failures red so users know they contribute 0 to the score.